add details about pooler auth (#2682)
timescale/docs: use-timescale/services/connection-pooling.md

application to open many short-lived connections, while the database opens few,
long-lived connections.

## User authentication

> By default, the pooler passes through authentication to the database, so you can
> keep using any custom users you already have set up without further
> configuration. You can also continue using the `tsdbadmin` user, if that is your
> preferred method. However, for some cases, you might need to add custom
> configurations for a pooler user, such as `statement_timeout`.

Reviewer: ```suggestion
By default, the poolers have authentication to the database, so you can use any
custom users you already have set up without further configuration. You can continue
using the `tsdbadmin` user if that is your preferred method. However, you might need
to add custom configurations for some cases such as `statement_timeout` for a pooler user.
```

<Procedure>

### Creating a new user with custom settings

> 1.  Log in to your database as the `tsdbadmin` user, and create a new role:

Reviewer: ```suggestion
1.  Log in to your database as the `tsdbadmin` user, and create a new role named `<MY_APP>` with the password as `<PASSWORD>`:
```

    ```sql
    CREATE ROLE my_app LOGIN PASSWORD '<NEW_PASSWORD>';
    ```

> 1.  Alter the role to change the setting for this user. This example changes the

Reviewer: ```suggestion
1.  To change the `statement_timeout` settings to 2 seconds for this user:

>    `statement_timeout` to 2 seconds:

Reviewer: ```suggestion
   
```

    ```sql
    ALTER ROLE my_app SET statement_timeout TO '2s';
    ```

> 1.  Connect on the pooler with the new user:

Reviewer: ```suggestion
1.  In a new terminal window, connect on the pooler with the new user `<MY_APP>`:
```

    ```bash
    ❯ PGPASSWORD=<NEW_PASSWORD> psql 'postgres://my_app@service.project.tsdb.cloud.timescale.com:30477/tsdb?sslmode=require'
    ```

    The output looks something like this:

    <CodeBlock canCopy={false}
    showLineNumbers={true}
    children={`
    psql (15.3 (Homebrew), server 15.4 (Ubuntu 15.4-1.pgdg22.04+1))
    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
    Type "help" for help.

>     tsdb=> select current_user;

Reviewer: This is a new step to check if you looked in as '<MY_APP>':

    ┌──────────────┐
    │ current_user │
    ├──────────────┤
    │ my_app       │
    └──────────────┘
    (1 row)

>     tsdb=> show statement_timeout;

Reviewer: This is another step. To check the `statement_timeout` setting soft `<MY_APP>` user:
